alphastats/loader/maxquant_loader.py

```diff
@@ -45,6 +46,23 @@ def __init__(
         self._set_filter_columns_to_true_false()
         self._read_all_column_names_as_string()
 
+        intensity_columns = [
+            col
+            for col in self.rawinput.columns
+            if intensity_column.replace("[sample]", "") in col
+        ]
+        if len(self.rawinput.dropna(subset=intensity_columns, how="all")) != len(
+            self.rawinput
+        ):
+            valid_id = re.compile(
+                "[A-Z]"
+            )  # Assuming that all valid protein ids would contain at least one letter.
+            self.rawinput = self.rawinput[
+                self.rawinput[self.index_column].apply(
+                    lambda x: bool(valid_id.match(x))
+                )
+            ]
```

Comment:
I don't think that is necessary. I don't know of any protein database that does not always have a letter in the identifiers.

Reply:
if it's that rare of an occasion I would even raise in this case

Reply:
This is by no means rare, unfortunatley. It is very common because the id columns at the end of the rows often become so long they crack the row limits. See the adjustments I made to the numbers in the tests.